graph/types/blockchain.go

```diff
@@ -26,6 +26,21 @@ func UnmarshalAddress(v interface{}) (common.Address, error) {
 	}
 }
 
+func MarshalBytes(b []byte) graphql.Marshaler {
+	return graphql.WriterFunc(func(w io.Writer) {
+		_, _ = io.WriteString(w, strconv.Quote(string(b)))
+	})
+}
+
+func UnMarshalBytes(b interface{}) ([]byte, error) {
```

Comment:
This looks very different from the other type. I would be surprised if this worked.